src/main/java/guides/hazelcast/quarkus/CommandController.java

```diff
@@ -21,38 +21,26 @@
     @ConfigProperty(name = "CONTAINER_NAME")
     private String containerName;
 
-
     @Inject
     HazelcastInstance hazelcastInstance;
 
-    private ConcurrentMap<String,String> retrieveMap() {
+    private ConcurrentMap<String, String> retrieveMap() {
         return hazelcastInstance.getMap("map");
     }
 
-    @GET
+    @POST
     @Path("/put")
     @Produces(MediaType.APPLICATION_JSON)
     public CommandResponse put(@QueryParam("key") String key, @QueryParam("value") String value) {
         retrieveMap().put(key, value);
-        return new CommandResponse(value,containerName);
+        return new CommandResponse(value, containerName);
     }
 
     @GET
     @Path("/get")
     @Produces(MediaType.APPLICATION_JSON)
     public CommandResponse get(@QueryParam("key") String key) {
         String value = retrieveMap().get(key);
-        return new CommandResponse(value,containerName);
-    }
-
-
-
-    HazelcastInstance createInstance() {
-        ClientConfig clientConfig = new ClientConfig();
-        clientConfig.getNetworkConfig().getKubernetesConfig().setEnabled(true);
-        clientConfig.getNetworkConfig().getKubernetesConfig().setProperty("service-name","hazelcast-cluster");
-        return HazelcastClient.newHazelcastClient(clientConfig);
```

Comment:
How did this k8s code end up being here? Weird but thanks for cleaning.

Reply:
My guess is you started with the k8s samples and then reused them here